clab/config.go

```diff
@@ -33,6 +33,8 @@ const (
 	hostNSPath = "__host"
 	// veth link mtu. jacked up to 65k to allow jumbo testing of various sizes
 	defaultVethLinkMTU = 65000
+	// containerlab's reserved OUI
+	clabOUI = "aa:c1:ab"
```

Comment:
love how `c1:ab` == `clab` in leetspeak. solid!

Reply:
yeah, and it's so easy to spot the interfaces managed by clab!